README.md

```diff
@@ -42,8 +42,7 @@ match '/saml/logout' => 'saml_idp#logout', via: [:get, :post, :delete]
 Create a controller that looks like this, customize to your own situation:
 
 ``` ruby
-class SamlIdpController
-  include SamlIdp::IdpController
+class SamlIdpController < SamlIdp::IdpController
```

Comment:
@meetme2meat you are correct, and I retracted here https://github.com/sportngin/saml_idp/pull/60#issuecomment-289142434